docs/content/faq.md

```diff
@@ -250,23 +249,32 @@ See [#1247][github-issue-1247] for an example.
 
 ### Common Errors
 
+Normally you will assign DMS a `hostname` such as `mail.example.com`. If you instead use a bare domain (_such as `example.com`_) or add an alias / account with the same value as your `hostname`, this can cause a conflict due to mail addressed to `hostname` being configured for system account mail only via Postfix `main.cf` setting `mydestination`.
+
+When this conflict is detected you'll find logs similar to this:
+
 ```log
-warning: connect to Milter service inet:localhost:8893: Connection refused
-# DMARC not running
-# => /etc/init.d/opendmarc restart
-
-warning: connect to Milter service inet:localhost:8891: Connection refused
-# DKIM not running
-# => /etc/init.d/opendkim restart
-
-mail amavis[1459]: (01459-01) (!)connect to /var/run/clamav/clamd.ctl failed, attempt #1: Can't connect to a UNIX socket /var/run/clamav/clamd.ctl: No such file or directory
-mail amavis[1459]: (01459-01) (!)ClamAV-clamd: All attempts (1) failed connecting to /var/run/clamav/clamd.ctl, retrying (2)
-mail amavis[1459]: (01459-01) (!)ClamAV-clamscan av-scanner FAILED: /usr/bin/clamscan KILLED, signal 9 (0009) at (eval 100) line 905.
-mail amavis[1459]: (01459-01) (!!)AV: ALL VIRUS SCANNERS FAILED
-# Clamav is not running (not started or because you don't have enough memory)
-# => check requirements and/or start Clamav
```

Comment:
Was it intentional to remove these errors?

Seems fine since there is no other context for them and they were [added from the old wiki](https://github.com/docker-mailserver/docker-mailserver/commit/5583ad27797c8ab5e0c433b9dbd264b88cb9f32a).